src/Devito.jl

```diff
@@ -35,6 +35,15 @@ end
 configuration(key) = PyDict(devito."configuration")[key]
 configuration() = PyDict(devito."configuration")
 
+function reversedims(arguments)
```

Comment:
I think that we don't need to make a new dictionary here.  Rather, we can construct a new named tuple.  Something like...

```julia
_reverse(argument::Tuple) = reverse(argument)
_reverse(argument) = argument

function reversedims(arguments)
    _arguments = collect(arguments)
    keys = first.(_arguments)
    values = @. _reverse(last(_arguments))    
    (; zip(keys, values)...)
 end
```